templates/cisco_asa_show_asp_table_vpn-context_detail.template

```diff
@@ -0,0 +1,37 @@
+Value VPN_CTX (\S+)
+Value PEER_IP (\d+.\d+.\d+.\d+)
+Value POINTER (\S+)
+Value STATE (\S+)
+Value FLAGS (\S+)
+Value SA (\S+)
+Value SPI (\S+)
+Value GROUP (\d+)
+Value PKTS (\d+)
+Value BAD_PKTS (\d+)
+Value BAD_SPI (\d+)
+Value SPOOF (\d+)
+Value BAD_CRYPTO (\d+)
+Value REKEY_PKT (\d+)
+Value REKEY_CALL (\d+)
+Value VPN_FILTER (\S+)
+
+Start
+  ^VPN\sCTX\s+=\s${VPN_CTX}
+  ^Peer\sIP\s+=\s${PEER_IP}
+  ^Pointer\s+=\s${POINTER}
+  ^State\s+=\s${STATE}
+  ^Flags\s+=\s${FLAGS}
+  ^SA\s+=\s${SA}
+  ^SPI\s+=\s${SPI}
+  ^Group\s+=\s${GROUP}
+  ^Pkts\s+=\s${PKTS}
+  ^Bad\sPkts\s+=\s${BAD_PKTS}
+  ^Bad\sSPI\s+=\s${BAD_SPI}
+  ^Spoof\s+=\s${SPOOF}
+  ^Bad\sCrypto\s+=\s${BAD_CRYPTO}
+  ^Rekey\sPkt\s+=\s${REKEY_PKT}
+  ^Rekey\sCall\s+=\s${REKEY_CALL}
+  ^VPN\sFilter\s+=\s${VPN_FILTER} -> Record
+  ^. -> Error
```

Comment:
```suggestion
  ^\s*
  ^. -> Error
```

Add catching an empty line just in case that shows up in some other model/version